src/services/api/site.js

```diff
@@ -1,8 +1,12 @@
-import { apiV1 } from 'services/config'
+import { apiV1, apiBase } from 'services/config'
+import { SITES } from 'common/constants'
 
 const site = {
-  get: async (sitename) => apiV1.get(`site/${sitename}`).json(),
-  getSites: async () => apiV1.get('site').json(),
+  get: async () => {
+    const response = await apiBase.get(SITES).json()
+    return response
+  },
+  getSites: async () => apiBase.get(SITES).json(),
```

Comment:
So from this diff it looks like `get` was originally retrieving a single site, and `getSites` was for the list. Now they both retrieve the list. I'm guessing this breaks site homepages and other things. It would probably be best to leave `get` the way it was, and use the `getSites` to retrieve the list data. Then you can do a separate PR for updating the `get` to retrieve the single site